aws/resource_aws_cloudfront_distribution_migrate_test.go

```diff
@@ -0,0 +1,52 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAwsCloudFrontDistributionMigrateState(t *testing.T) {
+	cases := map[string]struct {
+		StateVersion int
+		Attributes   map[string]string
+		Expected     map[string]string
+		Meta         interface{}
+	}{
+		"v0_1": {
+			StateVersion: 0,
+			Attributes: map[string]string{
+				"wait_for_deployment": "",
+			},
+			Expected: map[string]string{
+				"wait_for_deployment": "true",
+			},
+		},
+	}
+
+	for tn, tc := range cases {
```

Comment:
The variable names are a bit confusing here. I would stick with the std. i for index so that we don't have to think what it means sometime in the future. I would apply this thinking to any other variable that makes you think for more than two seconds.

Reply:
👍 will update these. We should consider updating the rest of the codebase too as this was mostly copied from elsewhere.